deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -129,6 +129,17 @@ externalDatabase:
   # -- The driverClassName of external database
   driverClassName: "org.postgresql.Driver"
 
+datasource:
+  profile: postgresql
+  config:
+    driver-class-name: org.postgresql.Driver
+    url: jdbc:postgresql://127.0.0.1:5432/dolphinscheduler
+    username: root
+    password: root
+    hikari:
+      connection-test-query: select 1
+      pool-name: DolphinScheduler
```

Comment:
It's incompatible with old db config, cuz ENV has higher priority.
Additionally, I recommend that `enable` option for different datasource should be provided here like before.

Reply:
I've already consider this situation. 
now application.yaml in docker are all configured like this way and env will overwrite this config.
so i kept old db config above so that they will become env to overwrite.
it is compatible already.

Reply:
https://github.com/apache/dolphinscheduler/issues/15924#issue-2265227765
Exactly, 
I will abandon that in next following plan. 
![image](https://github.com/apache/dolphinscheduler/assets/13224827/d5f9f6b4-981e-47df-a2fb-9361a4988fdf)

Reply:
sure, i will follow this guideline here

Reply:
> Got it. If you want to do this step by step, I suggest you in this PR, just define new configMap and do not modify `values.yaml`. In future PR, replace old db/registry config with new db/registry all together.
> 
> From my perspective, this way is clearer :D

Done.